tests/plugins/pastefromwordimage/manual/pastewordwithimage.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: bug, 4.8.0, 662, pastefromwordimage
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, pastefromwordimage, sourcearea, elementspath, newpage
```

Comment:
I would add `resize` plugin so you can enlarge editor to see whole pasted content.